### PR TITLE
OBPIH-1140 Document React components 

### PR DESCRIPTION
**Pull request by DominikaHatala**
Document components/methods/propTypes

### CODE REVIEW

**Comment by jmiranda** [comment]
Agree with @kkaczmarczyk. We can merge once we figure out how to include with the documentation.

File: README.md [hunk 1]
```diff
@@ -185,6 +185,15 @@ To run new frontend (React) tests type:
 ```
 npm test
 ```
+##### 13. React documentation
+Start a style guide dev server:
```
Comment (kkaczmarczyk) [inline README.md]: @jmiranda do you think you would like to keep this info here in general README or some place different?

Comment (jmiranda) [reply README.md]: README seems fine for now since the README is geared towards developers. I'll add some documentation to the readthedocs content later.

**Comment by kkaczmarczyk** [comment]
I am closing this - we do have included documentation here: https://github.com/openboxes/openboxes/pull/459
It required a bit conflict resolution

**Comment by kkaczmarczyk** [comment]
@jmiranda  I'd suggest using Github Pages for hosting of the resulting documentation

**Pull request closed without merging**